examples/performance/README.md

```diff
@@ -26,6 +26,10 @@
     * `cire-repeats-inv` (int, 1): control detection of dimension-invariants
     * `cire-mincost-inv` (int, 50): minimum cost of a dimension-invariant candidate
     * `cire-onstack` (boolean, False): control scope of temporary arrays (on the heap by default)
+  * Device-specific:
+    * `gpu-fit` (boolean, False): control movement of saved TimeFunctions between host and device
+    * `gpu-direct` (boolean, False): tells Devito a GPU-aware MPI distribution will be used
```

Comment:
Ambiguous - this is not what GPU DIrect means.

Reply:
Right, changing